### PR TITLE
Soen 132 reject unapproved users

### PR DESCRIPTION
**Pull request by jfv-2000**
# Description

Implements/fixes [SOEN-132](https://soen390team8.atlassian.net/jira/software/projects/SOEN/boards/1?selectedIssue=SOEN-132)

- New attribute RejectedFlag
- Reject button to remove patient from unapproved users list 
- notify rejected user by email
- Change pipeline yml file to recreate the .env file with GitHub Secrets

Important:
Add password of the email to the local .env file

Note:
-Commented bugged test case that is on the backlog to be fixed

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Unit testing

# Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
|image|image|

### CODE REVIEW

**Review by amqkv — APPROVED** [review]
Amazing implementation! Teared up while reading uwu

**Review by KelvinTrang — APPROVED** [review]
Goodjob mate

File: .github/workflows/pipeline.yml [hunk 1]
```diff
@@ -29,11 +29,16 @@ jobs:
           cd ../backend
           npm ci
 
+      - name: Create env file
+        run: |
+          touch ./backend/.env
+          echo ADMIN_EMAIL_SENDER_PASSWORD=${{ secrets.ADMIN_EMAIL_SENDER_PASSWORD }} > ./backend/.env
```
Comment (NicholasYip) [inline .github/workflows/pipeline.yml:35]: how many hours will this save?

File: frontend/components/navigation/navbar.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ export default function Navbar() {
 
     return (
         <>
-            <Box as="header" px={4} backgroundColor="white">
+            <Box zIndex={1000} as="header" px={4} backgroundColor="white">
```
Comment (NicholasYip) [inline frontend/components/navigation/navbar.tsx]: i don't like 1000, 69 or 420 seem like better choice

**Review by NicholasYip — APPROVED** [review]
you're cool

File: .github/workflows/pipeline.yml (hunk 1 above)
Comment (jfv-2000) [reply .github/workflows/pipeline.yml:35]: this exact line of code added 5 hours to my time spent debugging env files on pipeline XDXDXDXDXDXD

File: frontend/components/navigation/navbar.tsx (hunk 2 above)
Comment (jfv-2000) [reply frontend/components/navigation/navbar.tsx]: maybe we should do 69420 instead

**Review by stevenvuong00 — APPROVED** [review]
Great job! We appreciate the sacrifice of your sanity good sir.

**Pull request merged**